t/oid-info/oid

```diff
@@ -27,3 +27,8 @@ numeric		sha1:0123456789012345678901234567890123456789
 numeric		sha256:0123456789012345678901234567890123456789012345678901234567890123
 deadbeef	sha1:deadbeefdeadbeefdeadbeefdeadbeefdeadbeef
 deadbeef	sha256:deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef
+# collision-related constants
+t3404_collision	sha1:6bcd
+t3404_collision	sha256:0161
+t3404_collider	sha1:ac4f2ee
+t3404_collider	sha256:16697
```

Comment:
Instead of putting these in this file, we'd usually put them in a setup block in the test file itself, like so:

```sh
test_expect_success 'setup' '
	test_oid_cache <<-EOF
	# collision-related constants
	t3404_collision	sha1:6bcd
	t3404_collision	sha256:0161
	t3404_collider	sha1:ac4f2ee
	t3404_collider	sha256:16697
	EOF
'
```

That will load them only for the one particular test, similar to how we do it in other tests.

Reply:
Yes! I will do that.

Reply:
> Instead of putting these in this file, we'd usually put them in a setup block in the test file itself

@bk2204 please see the [interdiff](https://github.com/bk2204/git/compare/0a36b3828ed3ae079ec4c5b3f0337570d79456f6..db66920de0a1af729a736f270076023ede55605c) for my proposed fix